### PR TITLE
Remove underline from arrow used in search results pagination [YEP-48]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@bruskiza @miltontony tiny review please

File: tuneme/static/css/styles.css [hunk 1]
```diff
@@ -75,6 +75,7 @@ input[type='submit']:focus, a.button:focus {opacity:0.5;}
         .grey a.button.inverted {color:#828282; background:white; border-color:#828282;}
 
 
+.decoration {text-decoration: none;}
```
Comment (miltontony) [inline tuneme/static/css/styles.css]: It's better to use existing markup. `.step-links a {text-decoration: none;}`

Comment (smarzban) [inline tuneme/static/css/styles.css]: @miltontony if I use `a {text-decoration: none;}` then for feature phones all the links will be without underline

Comment (miltontony) [inline tuneme/static/css/styles.css]: @saeedmb8 `.step-links a` also, you can add it to `smart.css` if you only want it to apply to the smart version

**Comment by smarzban** [comment]
@miltontony done

**Comment by miltontony** [comment]
:+1:

**Pull request merged**